webapp/utils/format.ts

```diff
@@ -22,14 +22,14 @@ export const formatNumber = (value: number | string) =>
 export const formatFiatNumber = (value: number | string) =>
   fiatRounder(value, { shouldFormat: true })
 
-export const formatLargeFiatNumber = function (amount: number | string) {
-  // for less than one million, use the regular format.
-  if (Big(amount).lt(1_000_000)) {
-    return formatFiatNumber(amount)
+export const formatTVL = function (amount: number | string) {
+  if (Big(amount).lt(100_000)) {
+    // for less than 100k, return "< 100k"
+    const formatted = new Intl.NumberFormat('en-US', {
+      notation: 'compact',
+    }).format(100_000)
+    return `< $${formatted}`
```

Comment:
Can we just do this like...

```suggestion
    return `< $100K}`
```

Reply:
LOL yes! In hindsight, so obvious lol

Updated here [f31af5f](https://github.com/hemilabs/ui-monorepo/pull/970/commits/f31af5f7445663b48a210168451e7282f0bf3384)